src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticRawDataGroup.scala

```diff
@@ -36,86 +40,90 @@ case class ForestChangeDiagnosticRawDataGroup(umdTreeCoverLossYear: Int,
       totalArea,
       isUMDLoss && isTreeCoverExtent90
     ),
-    tree_cover_loss_primary_forest_yearly =
-      ForestChangeDiagnosticDataLossYearly.fill(
-        umdTreeCoverLossYear,
-        totalArea,
-        isPrimaryForest && isUMDLoss
-      ),
+    tree_cover_loss_primary_forest_yearly = ForestChangeDiagnosticDataLossYearly.fill(
+      umdTreeCoverLossYear,
+      totalArea,
+      isPrimaryForest && isUMDLoss
+    ),
     tree_cover_loss_peat_yearly = ForestChangeDiagnosticDataLossYearly.fill(
       umdTreeCoverLossYear,
       totalArea,
       isPeatlands && isUMDLoss
     ),
-    tree_cover_loss_intact_forest_yearly =
-      ForestChangeDiagnosticDataLossYearly.fill(
-        umdTreeCoverLossYear,
-        totalArea,
-        isIntactForestLandscapes2000 && isUMDLoss
-      ),
-    tree_cover_loss_protected_areas_yearly =
-      ForestChangeDiagnosticDataLossYearly.fill(
-        umdTreeCoverLossYear,
-        totalArea,
-        isProtectedArea && isUMDLoss
-      ),
-    tree_cover_loss_sea_landcover_yearly =
-      ForestChangeDiagnosticDataLossYearlyCategory.fill(
-        seAsiaLandCover,
-        umdTreeCoverLossYear,
-        totalArea,
-        include = isUMDLoss
-      ),
-    tree_cover_loss_idn_landcover_yearly =
-      ForestChangeDiagnosticDataLossYearlyCategory.fill(
-        idnLandCover,
-        umdTreeCoverLossYear,
-        totalArea,
-        include = isUMDLoss
-      ),
-    tree_cover_loss_soy_yearly =
-      ForestChangeDiagnosticDataLossYearly.fill(
-        umdTreeCoverLossYear,
-        totalArea,
-        isSoyPlantedAreas && isUMDLoss
-      ),
-    tree_cover_loss_idn_legal_yearly =
-      ForestChangeDiagnosticDataLossYearlyCategory.fill(
-        idnForestArea,
-        umdTreeCoverLossYear,
-        totalArea,
-        include = isUMDLoss
-      ),
-    tree_cover_loss_idn_forest_moratorium_yearly =
-      ForestChangeDiagnosticDataLossYearly.fill(
-        umdTreeCoverLossYear,
-        totalArea,
-        isIdnForestMoratorium && isUMDLoss
-      ),
-    tree_cover_loss_prodes_yearly = ForestChangeDiagnosticDataLossYearly.fill(
-      prodesLossYear,
+    tree_cover_loss_intact_forest_yearly = ForestChangeDiagnosticDataLossYearly.fill(
+      umdTreeCoverLossYear,
       totalArea,
-      isProdesLoss
+      isIntactForestLandscapes2000 && isUMDLoss
```

Comment:
I'm having trouble seeing why all these lines changed, was it because you fixed the indentation?

Reply:
Yeah, formatting fixes (indentation/mixed tab and spaces) and the line break style applied by VS code is slightly different (looks more Pythonic, below).  The [diff](https://github.com/wri/gfw_forest_loss_geotrellis/pull/171/files?diff=unified&w=1) looks a lot less noisier with the nifty 'ignore white space' setting.

existing:
``` 
    tree_cover_loss_intact_forest_yearly =
      ForestChangeDiagnosticDataLossYearly.fill(
        umdTreeCoverLossYear,
        totalArea,
        isIntactForestLandscapes2000 && isUMDLoss
      )
```
vs code:
```
    tree_cover_loss_intact_forest_yearly = ForestChangeDiagnosticDataLossYearly.fill(
      umdTreeCoverLossYear,
      totalArea,
      isIntactForestLandscapes2000 && isUMDLoss
    )
```